src/index.css

```diff
@@ -1,13 +1,43 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
+  padding: 0;
```

Comment:
w components stworz folder Header, wewnątrz umiesc Twoj component header.js i dodatkowo utworz plik scss ze stylami do tego komponentu. Jak jest troche wieksza aplikacja to lepiej miec style do kazdego komponentu osobno niż w jednym pliku

Reply:
done